jpo-asn-runtime/src/test/java/us/dot/its/jpo/asn/runtime/BaseSerializeTest.java

```diff
@@ -0,0 +1,91 @@
+package us.dot.its.jpo.asn.runtime;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.params.provider.Arguments;
+
+
+@Slf4j
+public abstract class BaseSerializeTest<T> {
+
+  private final static XmlMapper xmlMapper = new XmlMapper();
+  private final static ObjectMapper jsonMapper = new ObjectMapper();
+  private final Class<T> clazz;
+
+  public BaseSerializeTest(Class<T> clazz) {
+    this.clazz = clazz;
+  }
+
+  protected String toXml(T object) throws JsonProcessingException {
+    String str = xmlMapper.writeValueAsString(object);
+    log.info(str);

```

Comment:
question: there's a decent number of logs getting output when tests are ran, but are we gaining anything by using logging with the tests?

Reply:
I found them useful during development and testing, but they don't need to be so verbose every time the tests are run.  So I changed the level to debug and added simplelogger.properties defaulting to info level to suppress them by default.